data/json/items/ammo.json

```diff
@@ -464,8 +461,7 @@
     "volume": "2500 ml",
     "weight": "183 g",
     "ammo_type": "plasma",
-    "damage": 35,
-    "pierce": 14,
+    "damage": { "damage_type": "heat", "amount": 35, "armor_penetration": 14 },
```

Comment:
Just curious, how changing damage type to heat will affect for example shooting NPCs wearing some standard armor? Will it ignore it compleetly since armor has only bash and cut (and pierce) protection listed?

Reply:
Yess, changing damage type to heat will make it much more effective because very little has protection against heat, and things that do have low protection.

I can revert this change, but to me it didn't make sense that this did piercing damage.

Reply:
Actually, think about this more, I will back out the damage type change.